### PR TITLE
DOC: Significant simplifications to the README

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,26 +7,15 @@
...
-
-The main intent of ODL is to enable mathematicians and applied scientists to use different numerical methods on real-world problems without having to implement all necessary parts from the bottom up.
-This is reached by an `Operator` structure which encapsulates all application-specific parts, and a high-level formulation of solvers which usually expect an operator, data and additional parameters.
-The main advantages of this approach are that
-
-1. Different problems can be solved with the same method (e.g. TV regularization) by simply switching operator and data.
-2. The same problem can be solved with different methods by simply calling into different solvers.
-3. Solvers and application-specific code need to be written only once, in one place, and can be tested individually.
-4. Adding new applications or solution methods becomes a much easier task.
+Operator Discretization Library (ODL) is a Python library for fast prototyping focusing on (but not restricted to) inverse problems being developed at [KTH Royal Institute of Technology, Stockholm](https://www.kth.se/en/sci/institutioner/math) and [Centrum Wiskunde & Informatica (CWI), Amsterdam](https://www.cwi.nl).
```
Comment (kohr-h) [inline README.md]: Remove the KTH and CWI mention here and put them under "Funding".

File: README.md [hunk 2]
```diff
@@ -7,26 +7,15 @@
...
 
 Features
```
Comment (kohr-h) [inline README.md]: This whole section should IMO be replaced by "Highlights", not super-boring stuff like checking properties of norms.

Not quite sure if we should go for a "feature highlights" type list or rather examples of what you can do with ODL

File: README.md [hunk 3]
```diff
@@ -7,26 +7,15 @@
...
 ========
-- Efficient and well-tested data containers based on [NumPy](https://github.com/numpy/numpy) (default) or CUDA (optional)
+- Efficient and well-tested data containers based on [NumPy](https://github.com/numpy/numpy) or CUDA
 - Objects to represent mathematical notions like vector spaces and operators, including properties as expected from mathematics (inner product, norm, operator composition, ...)
-- Convenience functionality for operators like arithmetic, composition, operator matrices etc., which satisfy the known mathematical rules.
-- Out-of-the-box support for frequently used operators like scaling, partial derivative, gradient, Fourier transform etc.
```
Comment (kohr-h) [inline README.md]: Agree with deletion

File: README.md [hunk 4]
```diff
@@ -7,26 +7,15 @@
...
-Operator Discretization Library (ODL) is a Python library for fast prototyping focusing on (but not restricted to) inverse problems. ODL is being developed at [KTH Royal Institute of Technology, Stockholm](https://www.kth.se/en/sci/institutioner/math) and [Centrum Wiskunde & Informatica (CWI), Amsterdam](https://www.cwi.nl).
-
-The main intent of ODL is to enable mathematicians and applied scientists to use different numerical methods on real-world problems without having to implement all necessary parts from the bottom up.
-This is reached by an `Operator` structure which encapsulates all application-specific parts, and a high-level formulation of solvers which usually expect an operator, data and additional parameters.
-The main advantages of this approach are that
-
-1. Different problems can be solved with the same method (e.g. TV regularization) by simply switching operator and data.
-2. The same problem can be solved with different methods by simply calling into different solvers.
-3. Solvers and application-specific code need to be written only once, in one place, and can be tested individually.
-4. Adding new applications or solution methods becomes a much easier task.
```
Comment (kohr-h) [inline README.md]: I find this a bit too radical, at least one sentence that summarizes the main purpose of ODL (apart from the abstract "fast prototyping") should be there.

I'll try:

---

The main focus of *Operator Discretization Library* (ODL) is to enable research in inverse problems on realistic or real data. The framework allows to encapsulate a physical model into an `Operator` that can be used like a mathematical object in, e.g., optimization methods. Furthermore, ODL makes it easy to experiment with reconstruction methods and optimization algorithms for variational regularization, all without sacrificing performance.

For more details and an introduction into the inner workings of ODL, please refer to the [documentation](https://odlgroup.github.io/odl/).

File: README.md (hunk 2 above)
Comment (adler-j) [reply README.md]: If at all possible we should have actual code here. We need to think about a good example

File: README.md [hunk 5]
```diff
@@ -6,27 +6,16 @@
 
 ODL
 ===
+*Operator Discretization Library* (ODL) is a Python library that enable research in inverse problems on realistic or real data. The framework allows to encapsulate a physical model into an `Operator` that can be used like a mathematical object in, e.g., optimization methods. Furthermore, ODL makes it easy to experiment with reconstruction methods and optimization algorithms for variational regularization, all without sacrificing performance.
```
Comment (kohr-h) [inline README.md]: enables

**Comment by adler-j** [comment]
> Shall we merge this and make a separate issue for a code example?

Done that, lets think about a code example!

**Pull request merged**